lightspark-crypto-go/signing.go

```diff
@@ -0,0 +1,70 @@
+package lightspark_crypto
+
+// TODO(mhr): Add support for other OS.
+
+// #cgo darwin,amd64 LDFLAGS: -L${SRCDIR}/libs/darwin/amd64 -llightspark_crypto
+// #cgo darwin,arm64 LDFLAGS: -L${SRCDIR}/libs/darwin/arm64 -llightspark_crypto
+import "C"
+import (
+	"strings"
+
+	"github.com/lightsparkdev/lightspark-crypto-uniffi/lightspark-crypto-go/internal"
+)
+
+func GetMnemonicSeedPhrase(entropy []byte) ([]string, error) {
+	mnemonic, err := internal.MnemonicFromEntropy(entropy)
+	if err != nil {
+		return nil, err
+	}
+	defer mnemonic.Destroy()
+
+	return strings.Split(mnemonic.AsString(), " "), nil
+}
+
+func MnemonicToSeed(mnemonic []string) ([]byte, error) {
+	mnemonicObj, err := internal.MnemonicFromPhrase(strings.Join(mnemonic, " "))
+	if err != nil {
+		return nil, err
+	}
+	defer mnemonicObj.Destroy()
+
+	seed := internal.SeedFromMnemonic(mnemonicObj)
+	defer seed.Destroy()
+
+	return seed.AsBytes(), nil
+}
+
+func Ecdh(seedBytes []byte, derivationPath string, otherPubKey string) ([]byte, error) {
```

Comment:
@zhenlu what derivationPath am I supposed to pass here? It looks like the webhook only has a public_key

Reply:
According to @rmalonson's python signer it should be "m/0"

https://github.com/lightsparkdev/webdev/blob/main/sparkcore/sparkcore/node/signer.py#L196

I think I'm going to refactor the signer to take only one parameter.